.appveyor.yml

```diff
@@ -1,38 +1,28 @@
 version: '{build}'
-image: Visual Studio 2017
+image: Visual Studio 2015
```

Comment:
So these are tests. Can we/should we also have an example build (in a separate file), such that users don't take this as part of the example? Or maybe just a big, fat warning on top of these CI config files?

Reply:
I think the 17 -> 15 came from copying from python_example. These are "inspirational", which is why we have Travis, AppVeyor, and GHA. We should probably simplify both after the PR is merged, but part of the point is to show that what used to work, still works(*).

*: Actually the tests were completely broken on both platforms, but this is a minimal "fix" for those too, based directly on python_example.

Reply:
Hmmm, OK, yes. Let's see how many complaints we get of users that like to copy rather than take inspiration. Let's assume we'll be fine, and until then, we can stick with this! :-)